R/report.lavaan.R

```diff
@@ -12,23 +12,21 @@
 #' @examples
 #' library(report)
 #' \donttest{
-#' if (FALSE) {
-#'   # Structural Equation Models (SEM)
-#'   if (require("lavaan")) {
-#'     structure <- " ind60 =~ x1 + x2 + x3
+#' # Structural Equation Models (SEM)
+#' if (require("lavaan") && packageVersion("effectsize") >= "0.6.0.1") {
```

Comment:
@mattansb Will you be up for doing a patch release?

Because the `lavaan` methods are not going to work otherwise. We can get report on CRAN using this workaround, but that's not ideal for the users.

cf. https://github.com/easystats/effectsize/issues/410

Reply:
I can. Let's hope the CRAN gods won't be too angry...